### ISSUE TITLE
No such file or directory - pythonShellFile1110598151.py

### ISSUE THREAD

**Issue by Salamit** [extrabacon/python-shell#146]
Hi, 

I'm getting the following issue:
 Error: /Library/Frameworks/Python.framework/Versions/3.6/Resources/Python.app/Contents/MacOS/Python: can't open file 'var/folders/_z/18_yl0r12nn0zz5yyy0nx1kc0000gn/T/pythonShellFile1110598151.py': [Errno 2] No such file or directory

**Comment by Almenon** [comment]
Sorry for the delay - long day at work yesterday.  
How are you calling python-shell?

**Comment by Salamit** [comment]
Thanks for your response.

**Comment by Salamit** [comment]
Thanks for your response. Sorry for the delay too. I'm in Tokyo and just finished work. I'm calling it in meteor in my server.js file as follows

```
import {PythonShell} from 'python-shell';

PythonShell.runString('x=1+1;print(x)', null, function (err) {
    if (err) throw err;
    console.log('finished');
  });
```

**Comment by Almenon** [comment]
This might be a cross-platform bug in my code, because I tested it on windows and it worked.

But looking at the code I don't see any obvious bugs - I used the cross-platform variables tmpdir and sep, so it should work in any platform.

I'll test it in a different OS and try to reproduce the issue.

**Comment by Almenon** [comment]
@Salamit I replicated your issue on a mac.  The issue is fixed in the newest version I just released (v1.0.4)

**Comment by Almenon** [comment]
@eoaiu does it work in version 1.0.4?
